Article: Matrix Community Events and the Matrix Community Summit (#3359)
matrix-org/matrix.org: content/blog/2026/04/2026-04-20-community-events-and-summit.md

+++

> date = "2026-04-20T09:00:00Z"

Reviewer: (need to bump this)

Reviewer: ```suggestion
date = "2026-04-22T10:30:00Z"
```

title = "Matrix Community Events and the Matrix Community Summit"

[taxonomies]
author = ["Christian Paul", "Matrix Community Events"]
category = ["Events"]
+++

A big strength of Matrix is its popularity among IT hobbyists. These people discover and adopt the chat ecosystem early and often advocate for its introduction at their university, workplace and in their groups of friends. In order to bring these innovators and tinkerers together, a group called Matrix Community Events has been founded by passionate individuals.

> In-person events are an important part of the Matrix community in Europe. The annual conferences FOSDEM, FrOSCon and the Chaos Communication Congress are some events where many of our community members meet. These events allow us to exchange ideas across organisations, find new members on our quest to popularise Matrix and consolidate efforts on projects and spec proposals. To focus solely on Matrix, dedicated Matrix events take place, like meetups, workations ((group trips combining work and vacation)) and the annual Matrix Community Summit in Berlin.

Reviewer: i don't think we have a footnote feature, so maybe like this?
```suggestion
In-person events are an important part of the Matrix community in Europe. The annual conferences FOSDEM, FrOSCon and the Chaos Communication Congress are some events where many of our community members meet. These events allow us to exchange ideas across organisations, find new members on our quest to popularise Matrix and consolidate efforts on projects and spec proposals. To focus solely on Matrix, dedicated Matrix events take place, like meetups, workations (group trips combining work and vacation) and the annual Matrix Community Summit in Berlin.
```

<!-- more -->

> ## Matrix Conference or Matrix Community Summit?

Reviewer: https://matrix.org/branding/ says
```suggestion
## The Matrix Conference or Matrix Community Summit?
```

As an enthusiast, the answer to the question which event to participate in is likely both. The Matrix Conference and the Matrix Community Summit are annual events for people to learn from other experts and connect with others.

> The Matrix Conference is the official event of the Matrix Foundation and has been travelling to a different country each year since its inception in 2024. It's a conference with talks and networking in a professional setting for organisations, people from the public sector and community members. The presenters and attendees come from a wide variety of industries to share their lessons learned, projects and tips. A networking day before the conference and a hackathon day after the conference are additional ways for the community to use the momentum of such a big event.

Reviewer: https://matrix.org/branding/ says
```suggestion
The Matrix Conference is the official event of The Matrix Foundation and has been traveling to a different country each year since its inception in 2024. It's a conference with talks and networking in a professional setting for organisations, people from the public sector and community members. The presenters and attendees come from a wide variety of industries to share their lessons learned, projects and tips. A networking day before the conference and a hackathon day after the conference are additional ways for the community to use the momentum of such a big event.
```

The Matrix Community Summit is hosted by Matrix Community Events at c-base, a legendary hackerspace in Berlin. Since its inception in 2022, it caters to enthusiasts, students and power users who work with Matrix on a weekly basis. If you are involved in Matrix – or want to – this is for you. Instead of scheduled talks, expect free-form discussions and barcamps where people bring up their topics on the day. This year, the focus will be on coworking, outreach and community building through a space-themed drink mixing contest and Towel Day.

## Matrix Community Summit 2026

> Don't miss the chance to get your ticket now! There is barely one month left before it starts. The Matrix Community Summit 2026 starts on Thu, 21st May and ends on Mon, 25th May. Friday and Saturday are deep work days, so it might be the perfect coworking opportunity for your job. However, don't worry if you cannot make it on all five days.

Reviewer: I think it flows better to write out the weekdays?
```suggestion
Don't miss the chance to get your ticket now! There is barely one month left before it starts. The Matrix Community Summit 2026 starts on Thursday, 21st May and ends on Monday, 25th May. Friday and Saturday are deep work days, so it might be the perfect coworking opportunity for your job. However, don't worry if you cannot make it on all five days.
```

Isn't it weird to dayjob-work on the Saturday? maybe what's missing of should be the focus is deep work collaboration with contributors to one's project or WG.

Curious about the [Matrix Community Summit 2026](https://matrix-community.events/conferences/2026-summit.html)? The link contains all up-to-date info about the event.

> Learn more on the [website of Matrix Community Events](https://matrix-community.events/) and join the [matrix room of Matrix Community Events](https://matrix.to/#/#community-events:matrix.org) to get in touch with other members and to learn about the latest news.

Reviewer: ```suggestion
Learn more on the [website of Matrix Community Events](https://matrix-community.events/) and join the [Matrix room of Matrix Community Events](https://matrix.to/#/#community-events:matrix.org) to get in touch with other members and to learn about the latest news.
```

MCE also has a new fedi at https://mastodon.matrix.org/@community_events that I think we want ppl to follow for summit, stammtische and more event infos
